evap/static/css/evap.css

```diff
@@ -474,3 +474,7 @@ li.menu-indent {
 .table-striped tbody tr.sortable-ghost {
     background: #E5E5E5;
 }
+
+select {
+    width: 100%;
+}
```

Comment:
it's fine. other form-elements are styled explicitly and i didn't find any other occurrences which would now change their layout.